app/services/tickets/v1/create.rb

```diff
@@ -0,0 +1,36 @@
+module Tickets::V1
+  class Create
+    attr_accessor :title, :description, :ticket, :category_id, :department_id,
+                  :current_user, :priority, :resolver_id, :requester_id, :ticket_type
+
+    def initialize(params, user)
+      @current_user = user
+      @title = params[:title]
+      @description = params[:description]
+      @category_id = params[:category_id]
+      @department_id = params[:department_id]
+      @resolver_id = params[:resolver_id]
```

Comment:
resolver is assigned based on department right ?

Reply:
Done